### ISSUE TITLE
Add option to turn off stubgen

### ISSUE THREAD

**Issue by TheTripleV** [robotpy/mostrobotpy#16]

**Comment by virtuald** [comment]
There is already an environment variable in robotpy-build to suppress stubgen building.

**Comment by TheTripleV** [comment]
Right, but since there's actual tooling now, all the options should be configurable through the tooling. So I don't have to manually set environment variables every time I open a new terminal (and no, I don't want to globally set robotpy build options only to forget about em in the future)
